rpc/swagger/swagger.yaml

```diff
@@ -830,6 +830,14 @@ paths:
             type: number
             default: 30
             example: 30
+        - in: query
+          name: sort_oder
+          description: "Sort order (asc (for ascending) || desc (for descending))"
```

Comment:
can we add that this sorting is based on block height, people may get confused with other possible sorting methods

Reply:
Okay cool @marbar3778